extension/integration-tests/src/test/java/org/iris_events/it/rpc/RpcIntegrationTest.java

```diff
@@ -0,0 +1,33 @@
+package org.iris_events.it.rpc;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.io.IOException;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import jakarta.inject.Inject;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class RpcIntegrationTest {
+
+    @Inject
+    RpcClient rpcClient;
+
+    @Test
+    public void testRpcCall() throws InterruptedException, IOException, ExecutionException, TimeoutException {
+
+        var id = UUID.randomUUID();
+        final var rpcTestResponse = rpcClient.callRpc(id);
+
+        assertThat(id.toString(), CoreMatchers.is(rpcTestResponse.id()));
+    }
+}
```

Comment:
We should write this test bit more in spirit of extension tests:

```
public class IrisExtensionTest {
    @RegisterExtension
    static final QuarkusUnitTest config = new QuarkusUnitTest()
            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
                    .addClasses(TestConsumers.class)
                    .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml"));
@Inject 
 EventProducer producer;

    @Test
    public void testRPC() {
       var result = producer.sendAndRecive(...)
    }
}
```

Reply:
Implemented as above. Agreed it is much cleaner and easier to understand.